eng/testing/linker/project.csproj.template

```diff
@@ -11,6 +11,11 @@
     <!-- Workaround while there is no SDK available that understands the TFM; suppress unsupported version errors. -->
     <NETCoreAppMaximumVersion>99.9</NETCoreAppMaximumVersion>
     <_ExtraTrimmerArgs>{ExtraTrimmerArgs} $(_ExtraTrimmerArgs)</_ExtraTrimmerArgs>
+    <InterceptorsPreviewNamespaces>$(InterceptorsPreviewNamespaces);{InterceptorsPreviewNamespacesArgs}</InterceptorsPreviewNamespaces>
```

Comment:
Since we are referencing RDG by default, maybe this doesn't need to be a template arg. We could just have

```suggestion
    <InterceptorsPreviewNamespaces>$(InterceptorsPreviewNamespaces);Microsoft.AspNetCore.Http.Generated</InterceptorsPreviewNamespaces>
```

And then the test wouldn't need to set this arg anymore.